packages/bot-web-ui/src/components/dashboard/dashboard-component/cards.tsx

```diff
@@ -75,13 +76,13 @@ const Cards = ({
         {
             icon: 'IcBotBuilder',
             content: localize('Bot Builder'),
-            method: () => setActiveTab(1),
+            method: () => setActiveTab(DASHBOARD_TABS.BOT_BUILDER as number),
         },
         {
             icon: 'IcQuickStrategy',
             content: localize('Quick Strategy'),
             method: () => {
-                setActiveTab(1);
+                setActiveTab(DASHBOARD_TABS.BOT_BUILDER as number);
```

Comment:
Why do we need to cast it as a number? keeping at string should work right?

Reply:
it is a number

<img width="557" alt="Screenshot 2023-03-24 at 09 14 55" src="https://user-images.githubusercontent.com/103181650/227440342-09fbeaa7-3b86-47c8-85e9-efee29280797.png">

Reply:
<img width="557" alt="Screenshot 2023-03-24 at 09 17 30" src="https://user-images.githubusercontent.com/103181650/227440793-c3249ed4-6f14-4d0f-86fe-12975e903752.png">

Reply:
It has the proper equivalent numnber, do we still need to cast it to `number`

Reply:
<img width="876" alt="Screenshot 2023-03-24 at 09 20 00" src="https://user-images.githubusercontent.com/103181650/227441215-1524c565-bcb1-4b24-a556-c87061218118.png">

Reply:
we can create personal type for DASHBOARD_TABS, yeah, because it was type 
```
type TTabsTitleProps = {
    [key: string]: string | number;
};
```
for both object what is wrong
```
export const tabs_title: TTabsTitleProps = Object.freeze({
    WORKSPACE: 'Workspace',
    CHART: 'Chart',
});

export const DASHBOARD_TABS: TTabsTitleProps = Object.freeze({
    DASHBOARD: 0,
    BOT_BUILDER: 1,
    CHART: 2,
    TUTORIAL: 3,
});
```

Reply:
thank you that noticed that mess